### PR TITLE
README.md Update

### PR DESCRIPTION
**Pull request by zeel-codder**
#### Describe the changes you've made

Just On Function Name Problem In README.md 
it should like 
```python
x.send_file("file address")
```
not
```python
x.send_document("file address") 
```
In telegram Sample Code.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

#### Checklist

- [x] My code follows the code style of this project.
- [x] I have performed a self-review of my own code.
- [x] My change requires a change to the documentation.
- [x] I have read the **[CONTRIBUTING](https://github.com/rexdivakar/Notifly/blob/main/CONTRIBUTING.md)** document.
- [x] I have updated the documentation accordingly.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by rexdivakar — APPROVED** [review]

**Comment by rexdivakar** [comment]
Hey @zeel-codder  please replace the below token from temp.py file with {DISCORD_TOKEN} and i will approve your PR
image

**Comment by zeel-codder** [comment]
I change it. and still working on issue #31

File: temp.py [hunk 1]
```diff
@@ -3,8 +3,7 @@
 import tensorflow as tf
 
 ssl._create_default_https_context = ssl._create_unverified_context
-token = 'https://discord.com/api/webhooks/771253998313144321/5FNBsAi8-exyc-3rx' \
-        'LOYudQdQqMdpcaCOqCh6z1IX4ledG1oGyaONeIY1HDUns6qBZhW'
+token = 'https://discord.com/api/webhooks/885143691201773578/dJgoeeuYqhhCoBNC5nX5ZMBAlx05hKkXVAgSWrSUk-rPc-c7P4n0DL98yGdHwHbERnj4'
```
Comment (rexdivakar) [inline temp.py:6]: Please replace it with token = {DISCORD_TOKEN}

Comment (zeel-codder) [reply temp.py:6]: like with which token 🤔.

Comment (rexdivakar) [reply temp.py:6]: Just replace line 6 with below
`token = {DISCORD_TOKEN}`

Comment (zeel-codder) [reply temp.py:6]: I am asking what is DISCORD_TOKEN and form where I can find it?

Comment (rexdivakar) [reply temp.py:6]: It's a webhook from discord and I don't wanna expose my tokens anymore so I'm storing the token in the Github secrets

**Comment by zeel-codder** [comment]
hi, for issue (#31) there is a problem while we make requests in API. I add the exception to handle this issue.
Before
Screenshot from 2021-09-08 19-11-10

After
Screenshot from 2021-09-08 19-11-30

Thank you 👱

File: temp.py [hunk 2]
```diff
@@ -3,7 +3,8 @@
 import tensorflow as tf
 
 ssl._create_default_https_context = ssl._create_unverified_context
-token = 'https://discord.com/api/webhooks/885143691201773578/dJgoeeuYqhhCoBNC5nX5ZMBAlx05hKkXVAgSWrSUk-rPc-c7P4n0DL98yGdHwHbERnj4'
+token = 'https://discord.com/api/webhooks/771253998313144321/5FNBsAi8-exyc-3rx' \
```
Comment (rexdivakar) [inline temp.py]: Please replace this as requested

**Comment by SanchitJain123** [comment]
Great 👍

**Review by rexdivakar — APPROVED** [review]

**Pull request merged**